.rubocop.yml

```diff
@@ -0,0 +1,13 @@
+AllCops:
+  TargetRubyVersion: 2.6
```

Comment:
Maybe is better to use ruby in version 7.1, WDYT?

Reply:
It's possible to have any conflicts with pipefy-core using this version?

If not it's ok for me 👍🏾

Reply:
lol, I put the version o ES ahuahuauha

Sorry, I talk about the Ruby 3.1.0

Reply:
The pipefy core use ruby 2.6.6

Reply:
You are correct, the max version we could use here is 2.6.6